src/js/media-live-button.ts

```diff
@@ -80,6 +83,7 @@ class MediaLiveButton extends MediaChromeButton {
       ...super.observedAttributes,
       MEDIA_TIME_IS_LIVE,
       MEDIA_PAUSED,
+      MediaUIAttributes.MEDIA_LANG
```

Comment:
the constants above this line are destructured above, might as well do the same for `MediaUIAttributes.MEDIA_LANG`